src/types.ts

```diff
@@ -0,0 +1,31 @@
+import type { JsonRpcRequest } from 'json-rpc-engine';
```

Comment:
This file just has types in it. When `yarn build` is run, a `types.d.ts` file will be generated which has these types — all good — but a `types.js` will also be generated which is empty. I feel like this is how it's supposed to work, but it seems a bit weird to me. Not sure if there's another way we would prefer doing this or we've done this sort of thing before.

Reply:
Also, Istanbul thinks that there is no test coverage for this file. At the moment this is not breaking anything because I've lowered the coverage thresholds, but in the future I guess we should tell Istanbul to ignore this whole file?

Reply:
Could we make this a `.d.ts` file? That would get skipped by coverage steps, and would not generate an empty `.js` file.

Reply:
Hmm, I don't think so. Then `dist/createInfuraMiddleware.d.ts` tries to import `./types`, which is not included in the bundle, and I think that would cause errors for people.

Reply:
I meant in-place, like `src/types.d.ts`. But I see that that also wouldn't get included in the output. Hmm. We could add a build step to copy it over to the `dist` directory, but that seems a little janky. It would work though.

Otherwise, including this file as an empty JS module in the output, and skipping coverage for this file, seems OK to me. Not ideal but I don't see a great way of handling this right now.